oscovida/oscovida.py

```diff
@@ -144,7 +144,8 @@ def get_country_data_johns_hopkins(country: str,
     if len(tmp.shape) == 1:
         d = deaths.loc[country]
     elif len(tmp.shape) == 2:   # China, France, United Kingdom, ...
-        d = deaths.loc[country].sum()
+        tmp2 = tmp.drop(columns=['Province/State'])
```

Comment:
Do we need this intermediate `tmp2` object?

Reply:
Good point. The next commits try to tidy this up.

Reply:
> I don't actually understand the problem here. Could you explain? 🤷‍♂️

The table has a column "Province/State" which contains strings. The other columns are one for each date, and one each for longitude and latitude: all carry numbers. Thus ``sum`` works well on all columns apart from "Province/State". That's the one we need to remove (before calling ``sum``) to get rid of the new deprecation warning from pandas.

Reply:
Thanks for the explanation!